Automattic/jetpack
# Photon: remove unused method

## Proposed changes:

This hasn't been used since 2013 in 9b86a288d4e07c385109d931fcb31128d7e7622d.

We do not use that method on WordPress.com either.

We also do not need that method, since we already get Photonized images when we get images from the `Jetpack_PostImages` class (core functions like `wp_get_attachment_image_src` return a photonized image).

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings > Performance and enable the site accelerator.
* Go to Jetpack > Settings > Social and enable sharing buttons.
* Create a new post, and add a featured image to it.
* View the post on the frontend, and view source.
* Find the `og:image` meta. 
    * It should use Photon.
* Check your error logs; you should not see any notices.
